src/compas/datastructures/tree/tree.py

```diff
@@ -159,14 +160,18 @@ def descendants(self):
             for descendant in child.descendants:
                 yield descendant
 
-    def traverse(self, strategy="preorder"):
+    def traverse(self, strategy="depthfirst", order="preorder"):
         """
         Traverse the tree from this node.
 
         Parameters
         ----------
-        strategy : str, optional
-            The traversal strategy. Options are ``"preorder"`` and ``"postorder"``.
+        strategy : {"depthfirst", "breadthfirst"}, optional
+            The traversal strategy.
+            Default is ``"depthfirst"``.
```

Comment:
in my opinion you don't have to mention the default. this is already clear from line 163